Sources/MongoSwift/ReadPreference.swift

```diff
@@ -197,8 +197,8 @@ public struct ReadPreference: Equatable {
         if let maxStaleness = maxStalenessSeconds {
             guard maxStaleness >= MONGOC_SMALLEST_MAX_STALENESS_SECONDS else {
                 throw MongoError.InvalidArgumentError(
-                    message: "Expected maxStalenessSeconds to be >= " +
-                        " \(MONGOC_SMALLEST_MAX_STALENESS_SECONDS), \(maxStaleness) given"
+                    message: "Invalid \(MONGOC_URI_MAXSTALENESSSECONDS) \(maxStaleness): " +
```

Comment:
this was just an update to make it match the same error thrown when this happens in the connection string